python/spacewalk/satellite_tools/repo_plugins/yum_src.py

```diff
@@ -1243,6 +1243,10 @@ def set_download_parameters(self, params, relative_path, target_file=None, check
         params["timeout"] = self.timeout
         params["minrate"] = self.minrate
         params['proxies'] = get_proxies(self.proxy_url, self.proxy_user, self.proxy_pass)
+        params['urlgrabber_logspec'] = None
+        with cfg_component('server.satellite') as CFG:
+            if hasattr(CFG, "urlgrabber_logspec"):
+                params["urlgrabber_logspec"] = CFG.urlgrabber_logspec
```

Comment:
[nitpick] same here:
```suggestion
        with cfg_component('server.satellite') as CFG:
            params["urlgrabber_logspec"] = CFG.get("urlgrabber_logspec")
```